mobile-save-for-later/src/main/scala/com/gu/sfl/controller/FetchArticlesController.scala

```diff
@@ -24,6 +26,10 @@ class FetchArticlesController(fetchSavedArticles: FetchSavedArticles)(implicit e
            case e: IdentityServiceError =>  identityErrorResponse
            case e: MissingAccessTokenError => missingAccessTokenResponse
            case e: UserNotFoundError => missingUserResponse
+           case OktaOauthValidationError(e, InvalidOrExpiredToken) => oktaOauthError(e, StatusCodes.unauthorized)
+           case OktaOauthValidationError(e, MissingRequiredClaim(_)) => oktaOauthError(e, StatusCodes.forbidden)
```

Comment:
Would this error suggest a misconfiguration rather than a lack of permission?  If the request has the right scopes it should also have the right claims.

Reply:
A 400 might be better to give a hint that the token is badly formed if the claim is something that the endpoint is relying on

Reply:
updated it to use a 400!